manifests/init.pp

```diff
@@ -0,0 +1,176 @@
+# Sets up rkhunter to be run once per day
+#
+# @param minute_upd   Cron minute for update check
+# @param hour_upd     Cron hour for update check
+# @param monthday_upd Cron monthday for update check
+# @param month_upd    Cron month for update check
+# @param weekday_upd  Cron weekday for update check
+# @param minute       Cron minute
+# @param hour         Cron hour
+# @param monthday     Cron monthday
+# @param month        Cron month
+# @param weekday      Cron weekday
+#
+# @param package_ensure                The ensure status of packages to be managed
+#
+# @param rkhunter_conf_file            The path to the conf file
+#
+# @param rkhunter_conf_file_template   Template to use for the conf file
+#
+# @param check_for_updates             Check internet for definition updates
+#
+# @author https://github.com/simp/pupmod-simp-rkhunter/graphs/contributors
+#
+class rkhunter (
+  String[1]         $minute_upd                      = '0',
+  String[1]         $hour_upd                        = '0',
+  String[1]         $monthday_upd                    = '*',
+  String[1]         $month_upd                       = '*',
+  String[1]         $weekday_upd                     = '*',
+  String[1]         $minute                          = '10',
+  String[1]         $hour                            = '0',
+  String[1]         $monthday                        = '*',
+  String[1]         $month                           = '*',
+  String[1]         $weekday                         = '*',
+  String[1]         $package_ensure                  = 'latest',
```

Comment:
You should be using the simp_options setting for package_ensure and include simplib in the metadata.json file as a dependancy.   example: String $ensure = simplib::lookup('simp_options::package_ensure', { 'default_value' => 'installed' }),

Reply:
why installed as opposed to latest?

Reply:
@sharkbruhaha Standard safe practice

Reply:
roger